features/git-hack/hack_push_strategy_local.feature

```diff
@@ -0,0 +1,31 @@
+Feature: git hack: don't push branch to remote upon creation
+
+  As a developer starting work on a new private feature
+  I don't want git-hack to push my branch to the remote repo
+  So that the upstream repo is not aware of my work
+
+
+  Background:
+    Given my repo has an upstream repo
```

Comment:
In our tests origin is remote and upstream is used for terminology the repo the current is forked from